js/login.js

```diff
@@ -0,0 +1,34 @@
+(function($) {
```

Comment:
```suggestion
(function($) {
```
Ka jquery use garya pheri ?????